client/lobby/SelectionTab.cpp

```diff
@@ -868,7 +927,28 @@ void SelectionTab::parseSaves(const std::unordered_set<ResourcePath> & files)
 		}
 		catch(const std::exception & e)
 		{
-			logGlobal->error("Error: Failed to process %s: %s", file.getName(), e.what());
+			// asking for deletion of unsupported saves
```

Comment:
Also, this would generate this warning on every run.
Perhaps we should remember VCMI version on which this option was proposed and only ask again when player ugrades / downgrades VCMI?

Reply:
Still prefer asking every time. It's unnecessary to keep them a long time on HDD as them never will work again.

Only to give the player the possibility to finish a savegame by downgrading VCMI.